docs-release-notes.md

```diff
@@ -1,6 +1,13 @@
+# December 23, 2022
+
+- Device Sync:
+  - Add link to C++ docs
```

Comment:
```suggestion
  - Add link to C++ SDK (Alpha) docs
```

Reply:
We don't call it "C++ (Alpha)" in the Release Notes.